pkg/apis/workspaces/v1alpha1/commands.go

```diff
@@ -101,7 +101,7 @@ type VscodeConfigurationCommandLocation struct {
 
 type VscodeConfigurationCommand struct {
 	BaseCommand `json:",inline"`
-	Location    VscodeConfigurationCommandLocation `json:",inline"`
+	VscodeConfigurationCommandLocation `json:",inline"`
```

Comment:
Afaik this general GO convention isn't very consistent with K8S APIs practices (and bube-builder requirements): in many cases, the GO Struct field name should be consistent (apart from the 1st letter) with the json / yaml field name specified in the related tag.

Maybe we'd rather disable this GO linting rule since it seems it will not be possible to apply it everywhere, at least in the K8S API packages

Reply:
BTW if we think it is something that should still be done (though I don't know how due to kube-builder constraints), I'd rather do it globally in a distinct PR, as a distinct issue.

Reply:
In general, I've seen 
```
VSCodeConfigurationCommandLocation `json:vscodeConfigurationCommandLocation",inline"`
```
e.g. https://github.com/kubernetes/api/blob/b1350b9e3bc2ae69d6ea049ce1a637799a916e7d/auditregistration/v1alpha1/types.go#L175,

but I also don't know all the details and requirements.